### PR TITLE
llama index instrumentation

### PR DESCRIPTION
**Pull request by piotrm0**
See llama_example.ipynb for an example. Dashboard does not try to organize or present the llama index engine components in any special way.

### CODE REVIEW

File: trulens_eval/trulens_eval/tru_llama.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            root_method=TruLlama.query_with_record,
+            modules=LlamaInstrument.Default.MODULES,
+            classes=LlamaInstrument.Default.CLASSES,
+            methods=LlamaInstrument.Default.METHODS
+        )
+
+
+class TruLlama(TruModel):
+    """
+    Wrap a langchain chain for monitoring.
```
Comment (shayaks) [inline trulens_eval/trulens_eval/tru_llama.py:66]: This is a LlamaIndex model

File: trulens_eval/trulens_eval/tru_llama.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+    Arguments:
+    - chain: Chain -- the chain to wrap.
+    - More args in TruModel
+    - More args in LangChainModel
```
Comment (shayaks) [inline trulens_eval/trulens_eval/tru_llama.py:71]: LlamaIndexModel

**Review by shayaks — APPROVED** [review]

File: trulens_eval/trulens_eval/tru_llama.py (hunk 1 above)
Comment (piotrm0) [reply trulens_eval/trulens_eval/tru_llama.py:66]: fixed

File: trulens_eval/trulens_eval/tru_llama.py (hunk 2 above)
Comment (piotrm0) [reply trulens_eval/trulens_eval/tru_llama.py:71]: fixed

**Pull request merged**